src/main/kotlin/com/hj/leetcode/kotlin/problem808/Solution.kt

```diff
@@ -0,0 +1,52 @@
+package com.hj.leetcode.kotlin.problem808
+
+/**
+ * LeetCode page: [808. Soup Servings](https://leetcode.com/problems/soup-servings/);
+ */
+class Solution {
+    /* Complexity:
+     * Time O(1) and Space O(1) since the probability will reach 1.0-tolerance when n is
+     * greater than certain value n0;
```

Comment:
the causality is not clear

Reply:
Resolved at https://github.com/hj-core/hj-leetcode-kotlin/pull/114/commits/01713fc6a5374b1202a22a9738071be93cf1c4e6